requirements.txt

```diff
@@ -14,3 +14,5 @@ xml-python>=0.4.3
 packaging
 Cython
 lanms>=1.0.2
+imgaug
+Polygon3==3.0.9.1
```

Comment:
what is the difference of Polygon3 and `shapely.geometry.Polygon`，is is possible to use the later one without add new dependency?